### PR TITLE
Fix coinbase proof instructions

### PR DESCRIPTION
**Pull request by maxtaco**
- new settings page
- check that profile is public before going forward

### CODE REVIEW

**Comment by malgorithms** [comment]
Cool - hard for me to know if this works exactly right, without trying it, but the text + URL's LGTM. I'd say if it works for you, merge it.

File: go/libkb/proof_support_coinbase.go [hunk 1]
```diff
@@ -83,6 +84,29 @@ func (t CoinbaseServiceType) CheckUsername(s string) (err error) {
 	return
 }
 
+func coinbaseUserURL(s string) string {
+	return "https://coinbase.com/" + s
+}
+
+func coinbaseSettingsURL(s string) string {
```
Comment (malgorithms) [inline go/libkb/proof_support_coinbase.go]: looking at other functions here, should it be "CoinbaseSettingsURL()" not "coinbaseSettingsURL()" ? Just curious....I don't know what the preferred Go style is...

Comment (maxtaco) [inline go/libkb/proof_support_coinbase.go]: lowercase = not exported outside of the library

uppercase = exported outside of the library

weird, right?

**Pull request merged**